ffigen/pubspec.yaml

```diff
@@ -5,16 +5,8 @@ description: >-
          dart run ffigen --config macos.yaml
          dart run ffigen --config linux.yaml
 
-documentation: 
-publish_to: none
```

Comment:
wouldn't it be better to keep this so we explicitly state that this is not to be published